src/main/java/ch/njol/skript/conditions/CondIsTamed.java

```diff
@@ -0,0 +1,33 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Tameable;
+
+@Name("Is tamed")
+@Description("Check if a tameable entity is tamed (horse, parrot, cat, etc.).")
+@Examples({"send true if {_horse} is tamed",
+	"tame {_horse} if {_horse} is untamed"})
+@Since("INSERT VERSION")
+public class CondIsTamed extends PropertyCondition<Entity> {
+
+	static {
+		register(CondIsTamed.class, "(tamed|domesticated)", "entities");
+	}
+
+	@Override
+	public boolean check(Entity entity) {
+		if (entity instanceof Tameable)
+			return ((Tameable) entity).isTamed();
+		return false;
+	}
+
+	@Override
+	protected String getPropertyName() {
+		return "tamed";
+	}
+}
```

Comment:
```suggestion
	}

}
```
Skript's standardised formatting